test/Microsoft.AspNetCore.WebSockets.Internal.ConformanceTest/Autobahn/AutobahnExpectations.cs

```diff
@@ -13,16 +13,17 @@ public class AutobahnExpectations
         private Dictionary<string, Expectation> _expectations = new Dictionary<string, Expectation>();
         public bool Ssl { get; }
         public ServerType Server { get; }
+        public string Environment { get; }
```

Comment:
Good catch, it's something the WebSockets tests use but SignalR doesn't. Copy-pasta.